### PR TITLE
add new projects

### PR DESCRIPTION
**Pull request by cjabplanalp**

### CODE REVIEW

**Comment by ChristinaLK** [comment]
@matyasselmeci can you take a look at this?

File: projects/UWMadison_Gitter.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Department: Biostatistics and Medical Informatics
+Description: ''
```
Comment (ChristinaLK) [inline projects/UWMadison_Gitter.yaml:2]:
```suggestion
Description: 'https://www.biostat.wisc.edu/~gitter/'
```

File: projects/UWMadison_Curtin.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Department: Psychology
+Description: ''
```
Comment (ChristinaLK) [inline projects/UWMadison_Curtin.yaml:2]:
```suggestion
Description: 'https://arc.psych.wisc.edu/'
```

**Review by ChristinaLK — COMMENTED** [review]
adding descriptions to PR

**Comment by ChristinaLK** [comment]
@matyasselmeci can this be merged now?

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**